pouchdb-core/pouchdb-core.d.ts

```diff
@@ -64,12 +75,62 @@ declare namespace PouchDB {
                 ids: RevisionId[];
                 start: number;
             }
+
+            /** Attachments where index is attachmentId */
+            _attachments?: Attachments;
+        }
+
+        interface AttachmentResponse {
+            content_type: string;
+
+            /** Hash ? */
```

Comment:
@nolanlawson The CouchDB docs seem to indicate that this is always an `md5-`-prefixed base64-encoded MD5 hash.  Is that true?  Several examples in the PouchDB docs lack the `md5-` prefix.